Rakefile

```diff
@@ -44,30 +56,54 @@ task :benchmark do
   average_body = short_body * 1_000
   huge_body = average_body * 100
 
+  qs = 'don=quixote&quixote=don'
+
   short_request = TestSignableRequest.new(verb: 'PUT', request_url: '/', body: short_body)
+  qs_request = TestSignableRequest.new(verb: 'PUT', request_url: '/', body: short_body, query_string: qs)
   average_request = TestSignableRequest.new(verb: 'PUT', request_url: '/', body: average_body)
   huge_request = TestSignableRequest.new(verb: 'PUT', request_url: '/', body: huge_body)
 
+  v1_short_signed_request = mc.signed_v1(short_request)
+  v1_average_signed_request = mc.signed_v1(average_request)
+  v1_huge_signed_request = mc.signed_v1(huge_request)
+
+  v2_short_signed_request = mc.signed_v2(short_request)
+  v2_qs_signed_request = mc.signed_v1(qs_request)
+  v2_average_signed_request = mc.signed_v2(average_request)
+  v2_huge_signed_request = mc.signed_v1(huge_request)
+
   short_signed_request = mc.signed(short_request)
+  qs_signed_request = mc.signed(qs_request)
   average_signed_request = mc.signed(average_request)
   huge_signed_request = mc.signed(huge_request)
 
```

Comment:
Can you add a `puts` that tersely explains what short, average, huge, and qs mean?

Reply:
@ejennings-mdsol added.